modules/viqeoBidAdapter.js

```diff
@@ -0,0 +1,168 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {logError, logInfo, _each, mergeDeep, isFn, isNumber, isPlainObject} from '../src/utils.js'
+import {BANNER} from '../src/mediaTypes.js';
+import {Renderer} from '../src/Renderer.js';
+
+const BIDDER_CODE = 'viqeo';
+const VIQEO_ENDPOINT = 'https://ads.betweendigital.com/openrtb_bid';
+const RENDERER_URL = 'https://cdn.viqeo.tv/js/vq_starter.js';
+const DEFAULT_CURRENCY = 'USD';
+const DEFAULT_SSPID = 44697;
+
+function getBidFloor(bid) {
+  const {floor, currency} = bid.params;
+  const curr = currency || DEFAULT_CURRENCY;
+  if (!isFn(bid.getFloor)) {
+    return {floor: isNumber(floor) ? floor : 0, currency: curr};
+  }
+  const floorInfo = bid.getFloor({currency: curr, mediaType: BANNER, size: '*'});
+  if (isPlainObject(floorInfo) && isNumber(floorInfo.floor) && floorInfo.currency === curr) {
+    return floorInfo;
+  }
+  return {floor: floor || 0, currency: currency || DEFAULT_CURRENCY};
+}
+
+/**
+ * @type {BidderSpec}
+ */
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
```

Comment:
Your PR seems to suggest the bids you accept and respond with are always of type Outstream, this should be updated to reflect in your supportedMediaTypes in this PR and your prebid.github.io PR.

Reply:
fix it